human_experiments/scripts/run_experiment

```diff
@@ -30,7 +30,8 @@ __help() {
   echo "[10]: Starts Minecraft"
   echo "[11]: Starts the Client Map"
   echo "[12]: Opens the Final Survey"
-  echo "[13]: Ends the experiment (cleanup and backup phase)"
+  echo "[13]: Stop the Lab Recorder recordings and Kill the Lab Recorder Instances on CAT"
```

Comment:
Could you remove the unnecessary titlecasing here? That is, make the following changes:

- `Kill` -> `kill`
- `Instances` -> `instances`

Additionally, could you change `Lab Recorder` to `LabRecorder` to match the way it's written about in the LSL docs?

Reply:
Also, the cleanup phase was supposed to do more things like moving data from Cat to Tom for backup, and kill everything else that it's open not just the LabRecorder instances. That's why it was named more generically.

Reply:
@paulosoaresua - the cleanup phase has not been removed, it has just been moved to step 14 (see line 34 of the version of this file on this branch)